protocol/x/perpetuals/client/cli/query_perpetual.go

```diff
@@ -77,3 +80,20 @@ func CmdShowPerpetual() *cobra.Command {
 
 	return cmd
 }
+
+func CmdModuleNameToAddress() *cobra.Command {
```

Comment:
Can we add a quick test for this ([example](https://github.com/dydxprotocol/v4-chain/blob/25dbefe559a51b08cc06450a0b1c581fde0248dd/protocol/x/perpetuals/client/cli/query_all_liquidity_tiers_test.go#L16)), and very few this returns correct results for a few [module addresses](https://github.com/dydxprotocol/v4-chain/blob/25dbefe559a51b08cc06450a0b1c581fde0248dd/protocol/app/module_accounts_test.go#L80)

Reply:
This doesn't seem necessary given its a one liner which only prints an address. 
I'd also have to modify the code to return  a response for the test to be able to capture it which seems excessive for such a simple change

Reply:
`clitestutils` mock the i/o for the command, so there's no need to modify the code to return a response from the function.